diag_manager/fms_diag_object.F90

```diff
@@ -158,6 +159,7 @@ subroutine diag_obj_init(ob)
 end subroutine diag_obj_init
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 !> \description Fills in and allocates (when necessary) the values in the diagnostic object
+!
```

Comment:
This added line should be removed, since it's only 1 `!` doxygen won't read the line and it probably won't match the description to the routine.

Reply:
Done.